sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -1632,6 +1662,24 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
     }
   }
 
+  test("SELECT clause with star wildcard") {
+    withSQLConf(SQLConf.USE_NULLS_FOR_MISSING_DEFAULT_COLUMN_VALUES.key -> "true") {
+      withTable("t1") {
+        sql("CREATE TABLE t1(c1 int, c2 string) using parquet")
+        sql("INSERT INTO TABLE t1 select * from jt where a=1")
```

Comment:
We don't need to set SQLConf for this case.

Reply:
Done